index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;1,100&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Wiinnie's Portfolio</title>
+</head>
+<body>
+    <nav>
+        <div class="logo">
+            <a href="">John Doe</a>
+        </div>
+        <div class="menu-bar">
+            <img src="IMG1/icon.png" alt="Bars">
+        </div>
+    </nav>
+    <header>
+            <div class="header-container">
+                <h1>Winfred Edube</h1>
+                <h2>Software Developer</h2>
+            </div>
+            <div class="image">
+               
+            </div>
+        <div class="intro-container">
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me.</p>
+        </div>
+    </header>
```

Comment:
Please ensure that you stick to the design as much as possible (e.g. font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Styling: Active, Hover and Pressed for ALL buttons.

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/154397234-8b9104d9-4dfc-4cd3-a848-854715902c71.png) | ![image](https://user-images.githubusercontent.com/10332499/154397426-e4cdde22-0e0c-409b-aa84-1867f606b81b.png)